CHANGELOG.md

```diff
@@ -1,3 +1,76 @@
+# v 1.12.0 (2022-01-24)
+
+## OS Changes
+
+* Add patch to enable strict aliasing in dbus-broker ([#2730])
+* Add `/sys/firmware` to privileged mounts in host-ctr ([#2714])
+* Use user provided registry credentials for public.ecr.aws in host-ctr ([#2676])
+* Masked paths list built dynamically in host-ctr ([#2637])
+* Enable EFI option in systemd ([#2714])
+* Allow simple enums as map keys in datastore ([#2687])
+* Add method for fetching hostname in imdsclient ([#2647])
+* Add version 3 of network configuration to netdog ([#2596])
+* Add vlan structs for network configuration to netdog ([#2596])
+* Add bonding structs for network configuration to netdog ([#2596])
+* Migrator keeps only one intermediate datastore ([#2589])
+* Widen access to filesystem relabel in SELinux policy ([#2738])
+* Update hotdog to 1.05 ([#2728])
+* Update third party packages and dependencies ([#2717], [#2718])
+* Update eksd ([#2690], [#2693], [#2694], thanks @rcrozean)
+
+## Orchestrator Changes
+
+### Kubernetes
+
+* Add support for Kubernetes 1.25 variants ([#2699])
+* Restrict access to tls private key in kubelet ([#2639])
```

Comment:
The private key was already restricted; the actual effect was to remove the restrictions on reading public certificates to fix kube-bench: 
```suggestion
* Allow access to public kubelet certificates ([#2639])
```